cartographer/mapping/trajectory_node.h

```diff
@@ -67,8 +66,6 @@ struct TrajectoryNodes {
   std::vector<mapping::TrajectoryNode> trajectory_nodes;
 };
 
-proto::Trajectory ToProto(const std::vector<TrajectoryNode>& nodes);
```

Comment:
`cartographer_ros::WriteAssets` still calls this function. How will this be resolved?